app/models/concerns/sufia/work_stat_utils.rb

```diff
@@ -0,0 +1,34 @@
+module Sufia
+  module WorkStatUtils
+    def to_flots(stats)
+      stats.map(&:to_flot)
+    end
+
+    def convert_date(date_time)
+      date_time.to_datetime.to_i * 1000
+    end
+
+    private
+
+    def cached_stats(work_id, start_date, _method)
```

Comment:
Indent all methods after private/protected per the rails style: http://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#follow-the-coding-conventions